docs/admin/intro.md

```diff
@@ -6,3 +6,79 @@ title: Welcome
+
+## High Availability
+
+| Category                  | Details                                                                                                                                                                                                          |
+|:--------------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Redundancy & Failover** | <ul><li>Supported in clustered environments to ensure high availability and system reliability.</li><li>Automatic failover mechanisms help minimize downtime in case of hardware or software failures.</li></ul> |
+
+## Integrations
+
+| Category | Details |
+|:---------|:--------|
+| **Plugins & Extensibility** | <ul><li>Web: Extension system for adding custom functionality.</li><li>Server: Microservices architecture for modular and scalable server-side extensions</li></ul> |
+
+## APIs
+
+| API          | Description                                                                                                                                                       |
+|:-------------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **WebDAV**   | <ul><li>Standard file API for remote file management over HTTP/HTTPS.</li></ul>                                                                                   |
+| **GraphAPI** | <ul><li>Microsoft Graph API for managing cloud storage and collaboration spaces.</li></ul>                                                                        |
+| **gRPC**     | <ul><li>High-performance RPC framework for microservice communication.</li></ul>                                                                                  |
+| **OCS**      | <ul><li>API for programmatic file/folder sharing and management.</li></ul>                                                                                        |
+| **OCM**      | <ul><li>Open Cloud Mesh API for federated file and folder sharing between different file cloud platforms like Cernbox, ownCloud, Nextcloud or Seafile .</li></ul> |
+| **SSE**      | <ul><li>Server-sent event stream for real-time client notifications.</li></ul>                                                                                    |
+| **WOPI**     | <ul><li>Microsoft API for integrating Office apps with third-party storage, e.g., Collabora.</li></ul>                                                            |
+| **ICAP**     | <ul><li>Network protocol for offloading content scanning (e.g., antivirus).</li></ul>                                                                             |
```

Comment:
General wondering if the items inside the API section should be a list. IHMO not needed

Reply:
Same for compliance

Reply:
it would be semantically correct - if there is only one item, listing makes little sense. I had it like that before and decided differently because it looked more consistently like that. I'd stick with it as is.